pagerduty/resource_pagerduty_event_orchestration_path_router.go

```diff
@@ -294,7 +314,12 @@ func expandRouterActions(v interface{}) *pagerduty.EventOrchestrationPathRuleAct
 	actions := new(pagerduty.EventOrchestrationPathRuleActions)
 	for _, ai := range v.([]interface{}) {
 		am := ai.(map[string]interface{})
-		actions.RouteTo = am["route_to"].(string)
+		dra := am["dynamic_route_to"]
+		if !isNilFunc(dra) && len(dra.([]interface{})) > 0 {
+			actions.DynamicRouteTo = expandRouterDynamicRouteToAction(dra)
+		} else {
+			actions.RouteTo = am["route_to"].(string)
+		}
```

Comment:
**Annotation:** The `dynamic_route_to` action takes precedence over `route_to` but we can hopefully add validation for these actions to be mutually exclusive in the subsequent ticket (https://pagerduty.atlassian.net/browse/ORCA-4813).